tasks/scraping.go

```diff
@@ -0,0 +1,47 @@
+package tasks
+
+import (
+	"context"
+
+	"go-google-scraper-challenge/models"
+	"go-google-scraper-challenge/services/scraper"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/beego/beego/v2/task"
+)
+
+type ScrapingTask struct {
+	Tasker *task.Task
+}
+
+func (t *ScrapingTask) Name() string {
+	return "scraping"
+}
+
+func (t *ScrapingTask) GetTasker() *task.Task {
+	if t.Tasker == nil {
+		t.Tasker = task.NewTask(t.Name(), "0 * * * * *", perform)
+	}
+
+	return t.Tasker
+}
+
+func perform(_ context.Context) error {
+	result, err := models.GetFirstPendingResult()
+	if err != nil {
+		logs.Info("No pending result", err.Error())
+
+		return nil
```

Comment:
I think if the error is present, we should return it
```suggestion
		return err
```

Reply:
@junan Actually this is when there is no research to scrape so I don't think we should return the error 🤔  What do you think?